### PR TITLE
JS Local Storage

### PR DESCRIPTION
**Pull request by cherelemma**
# Preserve Data in the Browser

In this project work, we implemented storing contact information in the browser using localstorage API

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - The linters configuration for Javascript is not complete. Please update your `linters.yml` file and add the eslint section so the eslinter checks can run in the Github Pull Request. Please follow [this link](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @cherelemma 
I'm sure you put a lot of effort into this project but this PR is invalid because this is a pair programming activity but you've worked alone😓 

your partner needs to have some contribution in this project 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assests/javascript/main.js [hunk 1]
```diff
@@ -214,18 +212,55 @@ window.addEventListener('DOMContentLoaded', () => {
...
+}
+
+nameInput.addEventListener('change', onFormChange);
+emailInput.addEventListener('change', onFormChange);
+messageInput.addEventListener('change', onFormChange);
+
+// Load data in DOM on windows load
+window.onload = () => {
+  loadLocalStorage();
+};
```
Comment (GzimAsani) [inline assests/javascript/main.js:266]: - [ ] When filling the form I noticed these things:
- No matter what I put under name when I reload I get undefined under it
- The email and message get saved and when I reload I can see them. But if I leave them empty and reload I get the undefined word assigned to them(_Note: this happens only when I first load the page example: if I write something under email or message reload then delete and reload again, I will get them as empty, but if for a first time I leave them empty and reload I get undefined under word on their input_)

image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  **TEAM**                :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with local storage :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — APPROVED** [review]
Hi @cherelemma, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: Followed GitHub flow.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**